conans/test/functional/command/config_install_test.py

```diff
@@ -527,6 +530,26 @@ def get_zip():
         self.assertIn("Unzipping", self.client.out)
         http_server.stop()
 
+    def test_error_missing_origin(self):
+        path = self._create_zip()
+        self.client.run('config install "%s"' % path)
+        os.remove(path)
+        self.client.run('config install', assert_error=True)
+        self.assertIn("ERROR: Failed conan config install", self.client.out)
+
+    def test_list_remove(self):
+        path = self._create_zip()
+        self.client.run('config install "%s"' % path)
+        configs = json.loads(load(os.path.join(self.client.cache_folder, "config_install.json")))
+        self.assertIn("myconfig.zip", configs[0]["uri"])
+        self.client.run("config install --list")
+        self.assertIn("myconfig.zip", self.client.out)
+        self.client.run("config install --remove=0")
```

Comment:
Does the `conan config install --list` provide the index in the output so the user know how to identify each entry?

Reply:
I have been testing the branch and the index appears in the output